var/spack/repos/builtin/packages/armpl-gcc/package.py

```diff
@@ -123,6 +154,12 @@ class ArmplGcc(Package):
     conflicts("target=ppc64:", msg="Only available on Aarch64")
     conflicts("target=ppc64le:", msg="Only available on Aarch64")
 
+    conflicts("%gcc@:11.0", when="@22.1_gcc-11.2")
```

Comment:
Do you mean to say conflicts with GCC version `< 11.0`? Or `<= 11.0.x`?

Reply:
< 11.0.

Reply:
In that case it's `@:10`, this upperbound _includes_ any `10.x` version.